### PR TITLE
Add support for Huntsman v2 TKL

### PR DESCRIPTION
**Pull request by PrivateiFox**
Hi, I've added support for the Razer Huntsman v2 TKL in librazermacos, and I've noticed it's missing here as well.

Since the device is basically the same as the regular Huntsman v2 TKL, I've added support for it based on the features that device had.

I've tested the changes with a custom build -  the keyboard is detected properly, and all light effects seem to work just fine.

Thank you for all of your work on this program.

### CODE REVIEW

File: src/devices/huntsman_v2_tkl.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Razer Huntsman V2 TKL",
+  "productId": "0x026b",
+  "mainType": "keyboard",
+  "image": "https://assets3.razerzone.com/02qciF5cWDrRcrL9CxzbHC017No=/500x500/https%3A%2F%2Fhybrismediaprod.blob.core.windows.net%2Fsys-master-phoenix-images-container%2Fhb8%2Fhd7%2F9239675600926%2Fhuntsman-v2-tkl-500x500.png",
```
Comment (WrathofthePast) [inline src/devices/huntsman_v2_tkl.json:5]:
```suggestion
  "image": "https://dl.razerzone.com/src/5638/5638-1-en-v1.png",
```
This link is much shorter and the one openrazer used.

**Review by WrathofthePast — CHANGES_REQUESTED** [review]
Please review. If you are good with my suggested changes, lmk and I'll commit them.

File: src/devices/huntsman_v2_tkl.json (hunk 1 above)
Comment (PrivateiFox) [reply src/devices/huntsman_v2_tkl.json:5]: The suggested changes look good to me, thanks

**Review by WrathofthePast — APPROVED** [review]

**Comment by PrivateiFox** [comment]
I've changed the number of cols for the "ripple" and "wheel" effects. it now matches the cols setting of the Huntsman TE, and looks better.

**Pull request merged**